### ISSUE TITLE
Accessibility: Site is missing a “Skip to main” link at the top of the DOM

### ISSUE THREAD

**Issue by gmrabian** [coforma/swift-tech-challenge#213]
**Describe the bug**
Site is missing a “Skip to main” link at the top of the DOM

** Current behavior **
Tabbing through the site requires you to go through all the header content before main

**To Reproduce**
1. Go to the home page
2. Tab through the page

**Expected behavior**
There is a "Skip to main" option that speeds up navigation, especially for repeat users

## Definition of Done
[Definition of Done](https://github.com/coforma/swift-tech-challenge/blob/main/documentation/product/definition_of_done.md)
